src/executables/unload_schema.c

```diff
@@ -773,7 +780,15 @@ export_serial (print_output & output_ctx)
 
       output_ctx ("call [find_user]('%s') on class [db_user] to [auser];\n",
 		  db_get_string (&values[SERIAL_OWNER_NAME]));
-      output_ctx ("create serial %s%s%s\n", PRINT_IDENTIFIER (db_get_string (&values[SERIAL_NAME])));
+
+      serial_name_copy = strdup (db_get_string (&values[SERIAL_FULL_NAME]));
```

Comment:
Rather than using strdup(), it is more efficient to prepare a static array in advance and use memcpy().

Reply:
I will use this method while changing to a macro.